interp/constrintern.ml

```diff
@@ -2797,9 +2797,27 @@ let interp_named_context_evars ?(program_mode=false) ?(impl_env=empty_internaliz
 
 (** Local universe and constraint declarations. *)
 
+let known_universe_level_name evd lid =
+  try Evd.universe_of_name evd lid.CAst.v
+  with Not_found ->
+    let u = Nametab.locate_universe (Libnames.qualid_of_lident lid) in
+    Univ.Level.make u
+
+let known_glob_level evd = function
+  | GSProp | GProp ->
+    CErrors.user_err (Pp.str "Universe constraints cannot mention Prop or SProp.")
```

Comment:
This can be called from interp_known_level which is exposed in the mli, so not necessarily about constraints.

Reply:
Yes, but it's only used by constraint-manipulating functions.